### PR TITLE
Updated Dev and Test TF code to reflect Azure Configuration AB#7347

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Updated Dev and Test TF code to reflect Azure Configuration [AB#7347](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7347)

### CODE REVIEW

File: Azure/Dev/resource_virtual_network.tf [hunk 1]
```diff
@@ -184,7 +184,8 @@ resource "azurerm_application_gateway" "AppGate" {
       redirect_configuration,      
       url_path_map,     
       tags, # AGIC adds tags which need to be ignored. Can't seem to ignore the individual tags
-      ssl_certificate # see issue above 
+      ssl_certificate, # see issue above
+      waf_configuration.0 # see issue above
```
Comment (stuart-bass-cgi) [inline Azure/Dev/resource_virtual_network.tf]: can we please not ignore the waf rules, just change the tf to have them in the same order as they are in azure?

**Review by stuart-bass-bjss — CHANGES_REQUESTED** [review]

File: Azure/Dev/resource_virtual_network.tf (hunk 1 above)
Comment (gavinwakeley1) [reply Azure/Dev/resource_virtual_network.tf]: @stuart-bass-bjss Unfortunately, I tried that and set them to the same order as it is in Azure but that did not resolve the issue as it still reports them needing removing/re-adding - Terraform wants to 'own' the ordering which can only be done with an import or an apply. Once the change is made to the Azure TF module this line can be removed alongside the SSL one and an apply performed that will re-apply these settings exactly as they are but owned by Terraform.

Comment (gavinwakeley1) [reply Azure/Dev/resource_virtual_network.tf]: I've actually had another look and am applying the changes now

**Review by stuart-bass-bjss — APPROVED** [review]

**Comment by gavinwakeley1** [comment]
Merged to development

**Pull request merged**